software/main.py

```diff
@@ -19,43 +20,92 @@ class State(Enum):
     TESTCAMERA=5
     TMOTOR=6
     CALIBRATION=7
+    WAITING=8
+    NO_BALLS = 9
     
 
+class BasketColor(Enum):
+    BLUE=0
+    MAGENTA=1
+
+
+
 
 
 def main_loop():
-    state=State.FIND_BALL
-    spin = 13
+    state=State.WAITING
     debug=True
+    ref_cmds=False
+    basket_color = BasketColor.MAGENTA
 
-    basket_color=c.Color.MAGENTA
+    First_Ref=1
+    ref=client.Client()
+    if ref_cmds==True:
+        ref.start()
     cam=camera.RealsenseCamera(exposure=100) #defaulti peal on depth_enabled = True
     processor = image_processor.ImageProcessor(cam, debug=debug, color_config = "colors/colors.pkl")
     robot=movement.OmniRobot()
-    reso_x_mid=424
+    reso_x_mid=443

```

Comment:
Can be calculated based on camera center + a offset

Reply:
Changed it so that now the variable is calculated by adding cam.rgb_width/2 and a constant mid_offset.